Core/clim-basic/drawing/medium.lisp

```diff
@@ -459,7 +462,7 @@
 
 (defgeneric (setf graphics-state) (new-gs gs)
   (:method ((new-gs graphics-state) (gs graphics-state))
-    #+(or) "This is a no-op, so :after methods have primary method")
+    #+(or) "This is a no-op, so :after methods have the primary method.")
```

Comment:
Maybe "This is a no-op, but :after methods don't work without a primary method"?

Reply:
fixed